srv/modules/runners/populate.py

```diff
@@ -828,8 +822,14 @@ def _global(self):
         if not os.path.isdir(stack_dir):
             _create_dirs(stack_dir, self.root_dir)
         filename = "{}/global.yml".format(stack_dir)
+
+        __opts__ = salt.config.client_config('/etc/salt/master')
```

Comment:
...and this is reading `/etc/salt/master`

Reply:
Check the following lines which I pulled directly from https://docs.saltstack.com/en/latest/ref/clients/.  The short answer is that if I need access to modules from /srv/modules/modules, those are only available when using the master config.  I had to figure this out backwards after getting Jinja to work in normal state files.

That's why I think this different `__opts__` is worthy of discussion and another refactor.  It's unfortunate that Salt examples use `__opts__` for two different contexts.  If we are centralizing pieces like this in `__utils__`, I think we need precise names.

The other reason is the switch to the dunder calls.  Originally, I used the simplest examples of local.cmd.  IIRC, I found the Salt scopes confusing and likely could not get queries to work.